README.md

```diff
@@ -2,6 +2,19 @@
 
 a repo to practice clone, push, pull and handle merge conflicts.
 
+HEY ALL I NEED TO MAKE A PULL REQUEST TO DO SCREEN SHOTS FOR SOME WORK THING, ILL DELETE THIS WHEN IM DONE 
+
+BREAKING CHANGE
+
+COOL THING
+
+THING DONE USING OLD SYNTAX NOT OLD PATTERN
+
+THING IVE NEVER SEEN BEFORE
+
+SOMETHING THAT REQUIRES REVIEW
```

Comment:
Pause: implementing this change would cause a different thing in this other feature to become smaller. Can we check in with PM to make sure that this is intended?